### ISSUE TITLE
Please tell me how I start/run Auto-Sort-bookmarks for firefox

### ISSUE THREAD

**Issue by ckbeme** [eric-bixby/auto-sort-bookmarks-webext#87]
I feel stupid that I can't figure this out... How do I run Auto-Sort?

Thanks

**Comment by eric-bixby** [comment]
Once you've installed the add-on, there will be an icon added to the toolbar (3 yellow boxes and a blue arrow).  Click on the icon to show a menu: Settings, Configure Folders, Sort.
Select "Settings" to determine how you wish to sort and enable "Auto Sort" if you want sorting to happen when you make changes.
Select "Configure Folders" if you want to exclude certain folders from sorting.
Select "Sort" if you want to manually sort.

**Comment by ckbeme** [comment]
Thanks Eric,
for what it's worth, I just came back to FF after about 5 years on Chrome
cuz FF had slowed to a crawl.  After I'd been on Chrome for 3-4 years, it
started to slow down a lot and was a memory hog. Last straw was when they
made a stupid change to their Bookmarks manager. Everyone hates it. That
and Quantum made me come back to FF.
But your add-on isn't the first I had to really study to figure out how to
run. And I spent over 20 years in IT, I've been building my own systems for
ages, the last 5 or 6. So this was quite humbling that I wasn't able to
figure it out.

I appreciate your personal response.

Cathy

P.S. I've been really anxious to use your Auto Sort, can't wait.
